tests/ci_visibility/test_ci_visibility.py

```diff
@@ -6,15 +10,31 @@
 from ddtrace.ext import ci
 from ddtrace.internal.ci_visibility import CIVisibility
 from ddtrace.internal.ci_visibility.filters import TraceCiVisibilityFilter
+from ddtrace.internal.ci_visibility.git_client import CIVisibilityGitClient
+from ddtrace.internal.ci_visibility.git_client import CIVisibilityGitClientSerializerV1
 from ddtrace.internal.ci_visibility.recorder import _extract_repository_name_from_url
-from ddtrace.internal.writer.writer import Response
+from ddtrace.internal.utils.http import Response
 from ddtrace.span import Span
+from tests import utils
 from tests.utils import DummyCIVisibilityWriter
 from tests.utils import DummyTracer
 from tests.utils import override_env
 from tests.utils import override_global_config
 
 
+TEST_SHA = "b3672ea5cbc584124728c48a443825d2940e0ddd"
+
+
+@pytest.fixture
+def git_repo_empty(tmpdir):
+    yield utils.git_repo_empty(tmpdir)
+
+
+@pytest.fixture
+def git_repo(git_repo_empty):
+    yield utils.git_repo(git_repo_empty)
+
```

Comment:
Can we make the same fixtures as defined in `tests.contrib.pytest.test_pytest` import from here rather than define the same fixture twice?

Reply:
I tried to set that up, but the fact that they're fixtures makes that a bit tricky. Do you know how to put fixtures into shared library code?